lmdeploy/lite/quantization/awq.py

```diff
@@ -43,8 +43,10 @@
     'MixtralDecoderLayer': {
         'input_layernorm':
         ['self_attn.k_proj', 'self_attn.q_proj', 'self_attn.v_proj'],
-        'post_attention_layernorm':
-        ['block_sparse_moe.experts.{i}.w1', 'block_sparse_moe.experts.{i}.w3']
+        'post_attention_layernorm': [
+            'block_sparse_moe.gate', 'block_sparse_moe.experts.{i}.w1',
```

Comment:
Should we quantize `moe.gate`?

Reply:
It is skipped during quantization. This is for weight migration.